src/plugins/intel_cpu/src/transformation_pipeline.h

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/model.hpp"
+#include "low_precision/low_precision.hpp"
+#include "config.h"
+
+#include "itt.h"
+
+#include <memory>
+#include <vector>
+
+using namespace InferenceEngine;
+
+#define IE_CPU_PLUGIN_THROW(...) IE_THROW(__VA_ARGS__) << "CPU plugin: "
+
+namespace ov {
+namespace intel_cpu {
+
+class Transformations {
+public:
+    Transformations(const std::shared_ptr<ov::Model>& initialModel,
+                    const bool    enableLpt,
+                    const bool    enableSnippets,
+                    const bool    enableBF16,
+                    const bool    isLegacyApi,
+                    const Config& config)
+        : model(initialModel),
+          enableLpt(enableLpt),
+          enableSnippets(enableSnippets),
+          enableBF16(enableBF16),
+          isLegacyApi(isLegacyApi),
+          config(config) {}
+
+    void upToCpuSpecificOpSet();
+    void cpuSpecificOpSet(void);
+
+private:
+    std::shared_ptr<ov::Model> model;
+    const bool    enableLpt;
+    const bool    enableSnippets;
+    const bool    enableBF16;
+    const bool    isLegacyApi;
+    const Config& config;
+
+    void upToCpuSpecificOpSet_preLpt(const std::vector<ov::element::Type>& defaultPrecisions, const bool isLegacyApi);
```

Comment:
I would suggest to remove `preLpt` postfix from this method. I suppose it's better to have `common` postfix here because the vast majority of transformations are performed regardless of the model precision

Reply:
Was already discussed.
I also like the "common1" and "common2" instead of "preLpt" and "postLpt", since they are executed even for fp32 models where LPT is disabled.

Reply:
I think this is not about the method name but preLpt / postLpt naming itself (including filter names and corresponding IR names).
Now the dumped IR files (and filters) look like:
INT8:  preLpt, lpt, postLpt, snippets, cpuSpecificOpSet
FP32:  preLpt, postLpt, snippets, cpuSpecificOpSet

Instead we could have:
INT8:  common1, lpt, common2, snippets, cpuSpecificOpSet
FP32:  common1, common2, snippets, cpuSpecificOpSet

Reply:
Names for dumped IR files looks ok for me. My proposal it to leave current version.
However I was talking about method names inside transformation class. "upToCpuSpecificOpSet_preLpt" - looks weired. Lets just remove "upToCpuSpecificOpSet" prefix.

Reply:
Done